scripts/main.js

```diff
@@ -1 +1,38 @@
 console.log("[main.js] Test main.js");
+setTimeout(function () {
```

Comment:
Waiting for the [`load` event](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) to fire would work better here.

```suggestion
window.addEventListener("load", () => {
```